tests/src/JIT/HardwareIntrinsics/X86/Shared/AlternatingBinOpTest.template

```diff
@@ -162,6 +162,8 @@ namespace JIT.HardwareIntrinsics.X86
 
         public void RunBasicScenario_UnsafeRead()
         {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunBasicScenario_UnsafeRead));
```

Comment:
I'm not familiar with the coreclr test framework stuff, but since this is a "begin", do you want to also have an "end"?

Reply:
There is no "end scenario" method today, just "begin".

Basically it looks like:
* Begin Testcase
  * Begin Scenario
  * Begin Scenario
  * Begin Scenario
* End Testcase